core/job/service/job_service.go

```diff
@@ -90,6 +90,7 @@ type JobRepository interface {
 	GetAllByTenant(ctx context.Context, jobTenant tenant.Tenant) ([]*job.Job, error)
 	GetAllByProjectName(ctx context.Context, projectName tenant.ProjectName) ([]*job.Job, error)
 	UpdateState(ctx context.Context, jobTenant tenant.Tenant, jobNames []*job.Name, jobState job.State, remark string) error
+	SyncState(ctx context.Context, jobTenant tenant.Tenant, disabledJobs, enabledJobs []*job.Name) error
```

Comment:
instead of `disabledJobs` and `enabledJobs`, let's call them `disabledJobNames` and `enabledJobNames`. let's also apply it anywhere else to avoid confusion

Reply:
sure